V2/gameFuseUser.js

```diff
@@ -241,49 +314,155 @@
       }
     }
 
+    // async downloadAttributes(callback= undefined) {

```

Comment:
lets remove this or document why its commented if you might need it later

Reply:
cool, removed